nengo_ocl/clra_gemv.py

```diff
@@ -716,7 +718,7 @@ def many_dots_impl(p, items):  # noqa: C901
 
     segment_size = min(max_y_len, MAX_SEGMENT_SIZE)
     dot_block_size = min(
-        max(max_n_dots, 1), int(p.queue.device.max_work_group_size / segment_size)
+        max(max_n_dots, 1), int(p.queue.device.max_work_group_size / 2 / segment_size)
```

Comment:
Why this change?

Reply:
Oops. I was just testing it out and forgot to revert.